modules/apps/client-extension/client-extension-type-item-selector-web/src/test/java/com/liferay/client/extension/type/item/selector/web/internal/item/selector/CETItemSelectorViewDescriptorTest.java

```diff
@@ -65,12 +68,26 @@ public static void setUpClass() throws Exception {
 
 	@Test
 	public void testGetSearchContainer() throws PortalException {
+		GlobalCSSCET globalCSSCET1 = Mockito.mock(GlobalCSSCET.class);
+
 		Mockito.when(
-			_cetItemSelectorCriterion.getType()
+			globalCSSCET1.getScope()
 		).thenReturn(
-			ClientExtensionEntryConstants.TYPE_GLOBAL_JS
+			"company"
 		);
 
+		GlobalCSSCET globalCSSCET2 = Mockito.mock(GlobalCSSCET.class);
+
+		Mockito.when(
+			globalCSSCET2.getScope()
+		).thenReturn(
+			""
```

Comment:
Does it need explicitly to be "" or it can be a randomString()?

Reply:
`""` means that the value for that field was not set. Whoever did coded the CETs, forgot to use `null` instead of empty/default values.

Reply:
cool